syntax = "proto3";

message FrameResponse {
  int32 code = 1;
  enum Type {
    VIDEO = 0;//视频帧
    OUTLINE = 1;//轮廓帧
    BACKGROUND = 2;//背景帧
  }
  Type type = 2;
  bytes frame = 3;
  float position_x = 16;//物体横坐标
  float position_y = 17;//物体纵坐标
  int32 points_count = 18;//匹配特征点数
}
